examples/graphql/server-express.js

```diff
@@ -5,10 +5,33 @@ require('./tracer');
 const express = require('express');
 const { graphqlHTTP } = require('express-graphql');
 const buildSchema = require('./schema');
+const otel = require('@opentelemetry/api');
 
 const schema = buildSchema();
 
+const testMw = function (req, res, next) {
+  otel.context.with(
+    otel.propagation.setBaggage(
+      otel.context.active(),
+      otel.propagation.createBaggage({
+        'test': { value: 'testValue' },
+      })
+    ),
+    () => {
+      console.log('attached baggage', otel.propagation.getBaggage(otel.context.active()));
+      next();
+    }
+  );
+}
+
+const printMw = function (req, res, next) {
+  console.log('printMw baggage', otel.propagation.getBaggage(otel.context.active()));
```

Comment:
This is 'undefined'.

Reply:
Hi Amit - 
It looks like you're trying to report a bug, rather than contributing a code fix?  It would help us if you would [file an issue](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/new/choose) with your sample code rather than a pull request.